NF.csv

```diff
@@ -130,6 +130,11 @@ cAMP-Glo Max Assay,"The cAMP-Glo Max Assay from Promega is a bioluminescent assa
 Von Frey test,Assay applying electrical stimulus to assess pain in rodents.,"","",FALSE,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5592204/,assay,,"","",Von_Frey_Test,"",Assay,Assay,"","","",""
 targeted exome sequencing,"Also known as Clinical Exome Sequencing, Targeted Exome Sequencing is a subset of WES as it covers a limited number of genes, typically 3000 to 6000 genes.","","",FALSE,https://labassure.com/whole-exome-sequencing-vs-clinical-exome-sequencing/,assay,,"","",Targeted_Exome_Sequencing,"",Assay,Assay,"","","",""
 jumping library,"","","",FALSE,"",assay,,"","",Jumping_Library,Class,Assay,Method,"","","",""
+Child Behavior Checklist for Ages 1.5-5,"A standardized measure designed to detect behavioral and emotional problems in children 1.5-5 years old. The forms obtain parents', daycare providers', and teachers' ratings of 99 problem items plus descriptions of problems, disabilities, what concerns respondents most about the child, and the best things about the child. [ NCI ]","","",FALSE,"http://purl.obolibrary.org/obo/NCIT_C165712",assay,,"","",Child_Behavior_Checklist_for_Ages_1.5-5,Class,Assay,Method,"","","",""
```

Comment:
Not sure if these would too specific and should combine into just "Child Behavior Checklist"

Reply:
Thanks for the suggestion! I went with this because it sounds like they are two different measurements (and the data contributors confirmed they were using the ages 6-18 version, I added both before I knew which one.... :) )